.github/workflows/ci.yml

```diff
@@ -1,16 +1,22 @@
 name: CI
 
 on:
+  workflow_dispatch:
   push:
     branches:
-      - develop
+      - CI/multiple-jobs-pipeline
   pull_request:
     types: [ opened, synchronize, reopened ]
 
 jobs:
-  build:
-    # You must use a Linux environment when using service containers or container jobs
+  base-node-pipeline: 
+    uses: DEFRA/water-abstraction-orchestration/.github/workflows/base-node-pipeline.yml@develop
```

Comment:
This is clever. 

Presumably we can do the same across the other repos (UI/Returns/Permits/Import) etc? :D

Reply:
All should / can share the common pipeline. 

Best to discuss as a team on the common jobs and what node versions to test

Reply:
test setup looks a little different per repo so keep the test ci as a job in each repo.